eng/pipelines/templates/stages/archetype-cpp-release.yml

```diff
@@ -53,18 +52,17 @@ stages:
                     steps:
                       - checkout: none
                       - pwsh: Write-Host 'publish docs'
-                      - template: tools/clone-buildtools/clone-buildtools.yml@azure-sdk-build-tools
                       - pwsh: |
                           Get-ChildItem -Recurse $(Pipeline.Workspace)/docs/${{ artifact.Name }}
                         displayName: Output Visible Artifacts
-                      - template: tools/generic-blob-upload/publish-blobs.yml@azure-sdk-build-tools
+                      - template: ../../../common/pipelines/templates/steps/publish-blobs.yml
```

Comment:
You can use absolute paths by starting with `/` so `/eng/common/pipelines/templates/steps/publish-blobs.yml`